Simulation/TSoG_X1_Sim.m

```diff
@@ -75,7 +75,7 @@
     Results.Vy(i)    = StateVector(4);
     Results.Pitch(i) = StateVector(5);
     Results.Time(i)  = dt * (i - 1);
-    disp(ThrustModel(StateVector))
+    %disp(ThrustModel(StateVector))
```

Comment:
Looks like this is causing some merge conflicts, let's remove it for now since this will also spam the console with the value of Thrust while it's running

Reply:
Fixed this by reverting the changes